source/connect.txt

```diff
@@ -13,13 +13,75 @@ Connect to a Deployment
    :class: singlecol
 
 This page shows how to use the |mdb-shell| to connect to a MongoDB
-deployment.
+deployment. You can connect to a 
+`MongoDB Atlas cloud-hosted deployment <https://www.mongodb.com/docs/atlas>`__,
+connect to a local deployment, or connect to another remote host with
+|mdb-shell|.
 
 Prerequisites
 -------------
 
 .. include:: /includes/fact-connect-prereqs.rst
-   
+
+Install ``mongosh``
+~~~~~~~~~~~~~~~~~~~
+
+These procedures assume you have already installed ``mongosh``. For more 
+information about installing ``mongosh``, refer to 
+:ref:`mdb-shell-install`.
+
+Connect to a MongoDB Atlas Deployment
+-------------------------------------
+
+You can connect to your MongoDB Atlas deployment directly from your shell.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Get your Atlas Connection String
+
+      You need an Atlas connection string to connect from |mdb-shell|.
+      You can get the Atlas connection string in the Atlas UI.
+
+      Refer to the
+      `Find Your MongoDB Atlas Connection String 
+      <https://www.mongodb.com/docs/manual/reference/connection-string/#find-your-mongodb-atlas-connection-string>`__ 
+      guide for details.
+
+   .. step:: Set Your Database Credentials
+
+      If you haven't already 
+      `established a database user 
+      <https://www.mongodb.com/docs/atlas/tutorial/create-mongodb-user-for-cluster/>`__, 
+      you must set a username and password. You pass the username with 
+      the connection string to connect to Atlas. After you issue the 
+      connect command, the shell prompts for your password.
+
+   .. step:: Connect to MongoDB Atlas with ``mongosh``
+
+      Run the ``mongosh`` command with your connection string and options
+      to establish the connection.
+
+      The connection string includes the following elements:
+
+      - Your cluster name
+      - A hash
+      - A flag for the API version
+      - A flag for the username you want to use to connect
+
+      It resembles the following string:
+
+      .. code-block:: sh
+
+         mongosh "mongodb+srv://YOUR_CLUSTER_NAME.YOUR_HASH.mongodb.net/" --apiVersion YOUR_API_VERSION --username YOUR_USERNAME
+
+   .. step:: Learn More
```

Comment:
[suggestion] This might be better suited for a `.. note::` admonition, rather than a step since it seems more like additional information than a step everyone will take.